lib/tesla/adapter/mint.ex

```diff
@@ -315,7 +315,7 @@ if Code.ensure_loaded?(Mint.HTTP) do
 
         {:error, _conn, error, _res} ->
           if opts[:close_conn], do: {:ok, _conn} = close(conn)
-          {:error, "Encounter Mint error #{inspect(error)}"}
+          {:error, "Encountered Mint error #{inspect(error)}"}
```

Comment:
I am unsure if we should change the error, even when it is a misspelling; I am not sure who would be relying on it ...

Reply:
Or maybe not 0 https://github.com/Logflare/logflare/blob/dc9b17541eab4a7118cdc79adaf190b48cecdca3/lib/logflare_web/live/vercellogdrains_live/index.ex#L93

Reply:
Do you mind adding a TODO comment to change when we decide to break change?

Reply:
Very good idea, done that